### DISCUSSION TITLE
Localization causing error.

### DISCUSSION THREAD

**Discussion by ghost** [OrchardCMS/OrchardCore#15893] [Q&A] [answered]
I'm currently in the process of installing orchard in a Kubernetes cluster (linux node).  As started enabling features,  i noticed that whenever I enable localization,  I am no longer able to login. The message in browser is "your browser sent a request that this server could not understand"  I've checked the files under App_Data/logs to try and gain more info, but there isn't a log file generated for today.  What are the steps to getting to the bottom of an issue like this.  I'm not receiving the error locally using iis express

**Comment by hishamco** [comment]
We are unable to check without steps to reproduce, there should be logs, @Piedone did notice such an issue while working on the Kubernetes cluster while you own DotNest

**Reply by Piedone** [reply]
We haven't used Kubernetes, but I'll try to help.

**Comment by Piedone** [comment]
You have more than one node, right? Because for that, you'll need to start with this: https://github.com/OrchardCMS/OrchardCore/discussions/15149#discussioncomment-8213467. That might just fix this. Otherwise, the error is very generic, so I don't know what may be behind it; perhaps some cookie validation issue? But then even logging in shouldn't work.

**Reply by ghost** [reply, marked as answer]
Needed to update the annotations on my ingress.

nginx.ingress.kubernetes.io/proxy-body-size: 30m
nginx.ingress.kubernetes.io/proxy-buffer-size: 256k
nginx.ingress.kubernetes.io/proxy-buffering: 'on'
nginx.ingress.kubernetes.io/proxy-buffers-number: '4'
nginx.ingress.kubernetes.io/proxy-max-temp-file-size: 1024m
